doc/init.md

```diff
@@ -4,11 +4,11 @@ Sample init scripts and service configuration for bitcoind
 Sample scripts and configuration files for systemd, Upstart and OpenRC
 can be found in the contrib/init folder.
 
-    contrib/init/bitcoind.service:    systemd service unit configuration
-    contrib/init/bitcoind.openrc:     OpenRC compatible SysV style init script
-    contrib/init/bitcoind.openrcconf: OpenRC conf.d file
-    contrib/init/bitcoind.conf:       Upstart service configuration file
-    contrib/init/bitcoind.init:       CentOS compatible SysV style init script
+	contrib/init/bitcoind.service:    systemd service unit configuration
+	contrib/init/bitcoind.openrc:     OpenRC compatible SysV style init script
+	contrib/init/bitcoind.openrcconf: OpenRC conf.d file
+	contrib/init/bitcoind.conf:       Upstart service configuration file
+	contrib/init/bitcoind.init:       CentOS compatible SysV style init script
```

Comment:
Why is this changed?

Reply:
![image](https://user-images.githubusercontent.com/7684574/106870731-59717700-66c9-11eb-8104-db8339038f3e.png)

Reply:
That is a screenshot of the rendered changes of the second hunk. The first hunk (`contrib/init`) renders fine already.

Reply:
My bad—this is for tabs/spaces inconsistency consistency. Happy to go one way or the other.

Re: https://github.com/bitcoin/bitcoin/pull/21075#discussion_r570063070

Reply:
Yes, please don't use tabs. If it wasn't clear yet, we use spaces for indentation only in this project.

Reply:
Cool, thanks for the info. I've just replaced tabs for spaces in all other `doc/*.md` files too: [`e1604b3` (#21075)](https://github.com/bitcoin/bitcoin/pull/21075/commits/e1604b3d50dca3291a432be59cfd03c0e846e7b2)